BibBuddy/app/src/main/java/de/bibbuddy/NotesFragment.java

```diff
@@ -125,23 +129,30 @@ private void deselectNoteItems() {
     for (int i = 0; i < notesRecyclerView.getChildCount(); i++) {
       notesRecyclerView.getChildAt(i).setSelected(false);
     }
+
+    adapter.getSelectedNoteItems().clear();
```

Comment:
```suggestion
```
It is not necessary to clear that list, since the getSelectedNoteItems() is not counting manually but when used checking the current selected items nummer.

Reply:
True, thanks!